### PR TITLE
typescript-axios: add Set as language primitive

### LINKED ISSUE #6929: [BUG] [typescript-axios] uniqueItem Sets in api spec generate wrong typescript
##### Description
When generating client classes with generator template typescript-axios having properties with type array and flag "uniqueItems: true", the generated typescript model classes cannot be compiled. This only happens if template parameter "withSeparateModelsAndApi" is set to true.

##### openapi-generator version
 **5.0.0-SNAPSHOT**

##### OpenAPI declaration file content or url
```yaml
openapi: 3.0.2
info:
   title: Test API
   description: This is a test
   version: 0.0.1
   contact:
      name: Nobody
      email: nobody@noway.com
paths:
   /api/v1/test:
      get:
         operationId: getTest
         responses:
            "200":
               description: OK
               content:
                  application/json:
                     schema:
                        type: array
                        items:
                           $ref: "#/components/schemas/TestDTO"

components:
   schemas:
      TestDTO:
         type: object
         properties:
            uniqueItemList:
               uniqueItems: true
               type: array
               items:
                  type: integer
                  format: int64
            itemList:
               type: array
               items:
                  type: integer
                  format: int64
servers:
   - url: https://server.adress
     description: Server
```

#### Actual output
```typescript
// this import is wrong, Set is already a valid language primitive
import { Set } from './set';

/**
 * 
 * @export
 * @interface TestDTO
 */
export interface TestDTO {
    /**
     * 
     * @type {Set<number>}
     * @memberof TestDTO
     */
    uniqueItemList?: Set<number>;
    /**
     * 
     * @type {Array<number>}
     * @memberof TestDTO
     */
    itemList?: Array<number>;
}
```

#### Expected output
```typescript
/**
 * 
 * @export
 * @interface TestDTO
 */
export interface TestDTO {
    /**
     * 
     * @type {Set<number>}
     * @memberof TestDTO
     */
    uniqueItemList?: Set<number>;
    /**
     * 
     * @type {Array<number>}
     * @memberof TestDTO
     */
    itemList?: Array<number>;
}
```

##### Command line used for generation

**Used generator config:**
```json
{
	"basePackage": "src",
	"modelPackage": "src/model",
	"apiPackage": "src/api",
	"withSeparateModelsAndApi": true,
	"withInterfaces": true,
	"npmName": "test-api-client-typescript-axios",
	"snapshot": true
}
```

**Arguments:**
openapitools/openapi-generator-cli generate -i api-spec.yml -g typescript-axios -o out/generated/typescript-axios -c generator-config/config-typescript-axios.json

##### Steps to reproduce
1. Create api-spec.yml from above example.
2. Create config-typescript-axios.json from above example.
3. Call openapi generator with above parameters.

##### Related issues/PRs
No related issues found.

##### Suggest a fix
Add type Set to list of typescript language primitives.

### PR DESCRIPTION
**Pull request by Persi**
Suggested fix for #6929.

fixes #6929

### CODE REVIEW

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java [hunk 1]
```diff
@@ -140,6 +141,7 @@ public AbstractTypeScriptClientCodegen() {
         instantiationTypes.put("array", "Array");
 
         typeMapping = new HashMap<String, String>();
+        typeMapping.put("Set", "Set");
```
Comment (sarumont) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java:144]: @Persi You also need the lowercase mapping here: `typeMapping.put("set", "Set");`

Comment (Persi) [reply modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java:144]: Thank you for your feedback, i've added your mentioned line.

**Comment by macjohnny** [comment]
Please re-generate the samples/docs

**Comment by Persi** [comment]
> Please re-generate the samples/docs

How do I regenerate them? Maven does not seem to do that or I did not find the correct target ;)

**Comment by macjohnny** [comment]
https://github.com/OpenAPITools/openapi-generator/blob/master/.github/PULL_REQUEST_TEMPLATE.md

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

**Comment by Persi** [comment]
Thanks for your info, I've added a secondary mail to my account and updated the author of the commits. :)

**Comment by wing328** [comment]
Updated samples via f64a4d1a4f

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java [hunk 2]
```diff
@@ -140,6 +141,8 @@ public AbstractTypeScriptClientCodegen() {
...
+        typeMapping.put("set", "Set");
```
Comment (Xiphoseer) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java:145]: FYI: this line broke `typescript-angular` as described in https://github.com/OpenAPITools/openapi-generator/issues/8258#issuecomment-755409548.

More generally, it broke all the typescript generators that (for a response) just return the JSON object provided by a HTTP backend (which don't have runtime information on which JSON arrays correspond to schemas that have `uniqueItems` set). `Set<T>` has functions in it's prototype that `Array<T>` doesn't have (`add`, `clear`, `delete`, ...) and vice versa (`push`, `pop`, ...) so they are not interchangeable

**Pull request merged**